app/assets/javascripts/save_as.js

```diff
@@ -1,17 +1,22 @@
-/*globals document*/
+/*globals document,customPrompt*/
 $(document).on('page:change', function () {
     'use strict';
 
     function submitSaveAsForm() {
         var form = $(this).parents("form");
         form.attr("action", $(this).data('url'));
-        form.removeAttr("data-remote");
         form.submit();
     }
 
     $(".save_as").on("click", function (e) {
         e.preventDefault();
 
-        window.saveAll.submitForms(submitSaveAsForm.bind(this));
+        var nameInput = $("input#testing_ground_name");
+
+        customPrompt("Please change the name of the LES", nameInput.val(), function (newName) {
```

Comment:
Might I suggest "Please set a name for your new LES" – that would make it clear you're setting the name of the *new* one. The current wording might imply that the user is changing the name of the *old* LES.